.github/workflows/continuous-integration-workflow.yml

```diff
@@ -10,24 +10,20 @@ jobs:
         cmd: ['go_core_tests']
     name: Core Tests
     runs-on: ubuntu-latest
-    container:
-      image: smartcontract/builder:1.0.42
-      credentials:
-        username: ${{ secrets.DOCKER_READONLY_USERNAME }}
-        password: ${{ secrets.DOCKER_READONLY_PASSWORD }}
     env:
-      DATABASE_URL: postgres://chainlink@postgres:5432/chainlink_test?sslmode=disable
+      DATABASE_URL: postgresql://postgres:postgres@localhost:5432/chainlink_test?sslmode=disable
```

Comment:
Why has this changed?

Reply:
These changes to the CI files were pulled from PR #5389 . This change in particular is to specify the dummy username/password of the PG database that is running in the test instead of the passed in credentials to the builder image.